pow-migration/src/genesis/types.rs

```diff
@@ -39,21 +39,22 @@ pub enum Error {
 ///
 /// ```text
 ///     1                  2                  3                  4         5
-/// --- | ---------------- | ---------------- | ---------------- | ------- |
+/// --- | ---------------- | ---------------- | ---------------- | ------- | ----
 ///
 /// ```
 ///
 /// 1. Validator registration window start block.
 /// 2. Validator registration window end block. This block is also the pre-stake
 ///    registration window start.
-/// 3. Pre-stake registration window end block. This block is also the activation
-///    window start.
-/// 4. The final block in the PoW chain that will be taken as genesis block for the
-///    PoS chain. This block must have a block number that can be an election block
-///    number in the PoS chain.
+/// 3. Pre-stake registration window end block.
+/// 4. The candidate block in the PoW chain that will be taken as genesis block for the
+///    PoS chain.
 /// 5. This is a block whose block number is a number of confirmations away from
 ///    the final block described in 4.
 ///
+/// Note that after 5 there are some other blocks that are considered in the migration
```

Comment:
I don't understand this note.
After 5 is when we start the genesis generation process

Reply:
Removed it. It is going to be removed anyway in my next set of changes